dipy/viz/colormap.py

```diff
@@ -255,3 +261,52 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
```

Comment:
I respectfully disagree. The colormap is supposed to help derive the information from the data, but this one hides and distorts information.